ebpf/event_common.go

```diff
@@ -27,12 +27,21 @@ func (c ConnectionType) String() string {
 // ConnectionFamily will be either v4 or v6
 type ConnectionFamily uint8
 
+// ConnectionDirection indicates if the connection is incoming to the host or outbound
+type ConnectionDirection uint8
+
 const (
 	// AFINET represents v4 connections
 	AFINET ConnectionFamily = 0
 
 	// AFINET6 represents v6 connections
 	AFINET6 ConnectionFamily = 1
+
+	// INCOMING represents connections inbound to the host
+	INCOMING ConnectionDirection = 0
```

Comment:
Could we declare all those constants only in the C code and import them here ?
Like
```suggestion
	INCOMING ConnectionDirection = C.INCOMING
```